src/components/types.tsx

```diff
@@ -40,3 +41,26 @@ export type props = {
   navigation: ProfileScreenNavigationProp;
   route: ProfileScreenRouteProp;
 };
+
+export type TaskPopupProps = {
+  testID?: string;
+  item: number;
+  modalVisible: boolean;
+  setModalVisible: Dispatch<SetStateAction<boolean>>;
+};
+
+export type LoginScreenProps = StackScreenProps<
+  RootStackParamList,
+  'LoginScreen'
+>;
+
+export type task = {
```

Comment:
Should this type be named `Task` instead of `task`?

Reply:
Updated